website/source/docs/connect/mesh_gateway.html.md

```diff
@@ -24,6 +24,16 @@ Each mesh gateway needs three things:
 2. General network connectivity to all services within its local Consul datacenter.
 3. General network connectivity to all mesh gateways within remote Consul datacenters.
 
+Mesh gateways also require that your Consul datacenters are configured correctly:
+
+- You'll need to use Consul version 1.6.0-beta1.
```

Comment:
beta2 will work too. Should we just say 1.6.0 here maybe? Not a big deal regardless but if we say 1.6.0 then we wont need to update this again and I think its specific enough that someone would know that they can use 1.6.0 beta.

Reply:
```suggestion
- You'll need to use Consul version 1.6.0.
```